addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
I've never been a fan of nested destructuring. I don't think it fits with the reason why we were doing the destructuring of `Ember` to begin with. If I'm not mistaken, we were doing it b/c we expected ember to eventually move to named imports and 
```js
import Ember from 'ember'
const {Components, computed, run} = Ember
```
could easily be converted to
```js
import {Component, computed, run} from 'ember'
```

I don't believe we'd want
```js
import Ember from 'ember'
const {
  Component,
  computed: {
    readOnly
  },
  run
} = Ember
```
to become 
```js
import {
const {
  Component,
  computed: {
    readOnly
  },
  run
}  from 'ember'
```
At least, I've never seen that kind of destructuring on an `import` (if it's even supported) and I'd much rather see
```js
import {Component, computed, run} = Ember
const {readOnly} = computed
```
That's why, if we want to destructure one of the children of `Ember`, which has it's own downsides of doing (like making it harder to stub it in tests), I propose we do it like so:
```js
const {Component, computed, run} = Ember
const {readOnly} = computed
```